### DISCUSSION TITLE
Create file if it doesn't exist yet when using the "Open source file key"

### DISCUSSION THREAD

**Discussion by jamesvclements** [tadashi-aikawa/obsidian-various-complements-plugin#181] [General]
Thank you for building this! I've been using Obsidian for months and typing `[[` was starting to drive me crazy, the internal link complements are everything I wanted. 

It'd be awesome if the `Open source file` command created the file if it doesn't exist yet

**Comment by tadashi-aikawa** [comment]
Hi, @jamesvclements :)

> It'd be awesome if the Open source file command created the file if it doesn't exist yet

Sorry, I couldn't imagine the situation in your description. Could you tell me more details?
